packages/wallets/src/features/cashier/modules/Transfer/hooks/useSortTransferAccounts.ts

```diff
@@ -0,0 +1,91 @@
+import { useMemo } from 'react';
+import { getMarketType } from '../helpers';
+import { TAccount, TAccountsList } from '../types';
+
+const useSortTransferAccounts = (accounts: TAccountsList) => {
```

Comment:
"useSort**ed**TransferAccounts" might be better, 
"useSortTransferAccounts" does suggest that its a hook returning some sort of action to sort acount, and not just "sorted" accounts, 

basically, 
"useSort", "useModify", "useSubsribe" - naming for hooks returning actions
"useSorted", "useModified", "useSubscribed" - naming for hooks returning data